stingray/pulse/fftfit.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+from scipy.optimize import minimize, brentq
+
+
+def find_delay_with_ccf(amp, pha):
+    nh = 32
+    nprof = nh * 2
+    CCF = np.zeros(64, dtype=np.complex)
```

Comment:
I think PEP8 suggests that variables, functions and methods are all lower-case, and only classes have uppercase characters. I quite like not having random capitalized variables in the code, unless there's a really compelling reason for them, but I'm also not going to die on that hill. Thoughts?

Reply:
Done